actions/receive/action.yaml

```diff
@@ -64,7 +115,12 @@ runs:
         ref: ${{ inputs.benchmarking_ref }}
         path: ${{ inputs.benchmarking_path }}
     - name: Run benchmarks
-      run: touchstone::run_script("touchstone/script.R")
+      run: |
```

Comment:
Are you sure this is necessary? I think these envs are inherited in the R process? Otherwise, how would touchstone have worked in the past?

Reply:
That is necessary as the[ `issue_comment`-event](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issue_comment) does not receive the PR ref via it's payload unlike the [`pull_request`-event](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request).